### ISSUE TITLE
Quantization 

### ISSUE THREAD

**Issue by aminsabetsarvestani** [pytorch/vision#1601]
How can I wrap quantized operation cat in the following code from rpn.py? I am replacing torch.cat  with 
cat =nn.quantized.FloatFunctional()
cat.cat(box_cls_flattened, dim=1).reshape(-1, C)
 but I get the kernel didn't find dispatch to for operator error

def concat_box_prediction_layers(box_cls, box_regression):
    box_cls_flattened = []
    box_regression_flattened = []
    # for each feature level, permute the outputs to make them be in the
    # same format as the labels. Note that the labels are computed for
    # all feature levels concatenated, so we keep the same representation
    # for the objectness and the box_regression
    for box_cls_per_level, box_regression_per_level in zip(
        box_cls, box_regression
    ):
        N, AxC, H, W = box_cls_per_level.shape
        Ax4 = box_regression_per_level.shape[1]
        A = Ax4 // 4
        C = AxC // A
        box_cls_per_level = permute_and_flatten(
            box_cls_per_level, N, A, C, H, W
        )
        box_cls_flattened.append(box_cls_per_level)

        box_regression_per_level = permute_and_flatten(
            box_regression_per_level, N, A, 4, H, W
        )
        box_regression_flattened.append(box_regression_per_level)
    # concatenate on the first dimension (representing the feature levels), to
    # take into account the way the labels were generated (with all feature maps
    # being concatenated as well)
    box_cls = **torch.cat**(box_cls_flattened, dim=1).reshape(-1, C)
    box_regression = **torch.cat**(box_regression_flattened, dim=1).reshape(-1, 4)
    return box_cls, box_regression

**Comment by helmut-vineland** [comment]
was this ever answered?

**Comment by fmassa** [comment]
Quantization for faster rcnn is not yet supported in torchvision
